java/src/jmri/util/ColorUtil.java

```diff
@@ -53,41 +53,46 @@ public static String colorToColorName(@Nullable Color color) {
         return colorToHexString(color);
     }
 
+    /**
+     * @param string Either a hexidecimal representation of the rgb value of a 
+     * color or a color name defined in jmri.NamedBeanBundle.properties. 
+     */
     public static Color stringToColor(String string) {
-        switch (string) {
-            case "black":
+        try {
+            return Color.decode(string);
+        } catch(NumberFormatException nfe) {
+            if(string.equals(Bundle.getMessage("ColorBlack"))) {
                 return Color.black;
-            case "darkGray":
```

Comment:
If these were colors were defined as public static fields, you could use the case statement instead of repeatedly querying a resource bundle.